### ISSUE TITLE
error creating message for Grafana Loki: json: unsupported type

### ISSUE THREAD

**Issue by gregwebs** [cosmo0920/fluent-bit-go-loki#5]
There is some interesting recursion going on here with the error. When I look at the logs for fluent-bit I the error messages keep getting bigger.

```
error creating message for Grafana Loki: json: unsupported type: map[interface {}]interface {}error creating message for Grafana Loki: json: unsupported type: map[interface {}]interface {}error creating message for Grafana Loki: json: unsupported type: map[interface {}]interface {}error creating message for Grafana Loki: json: unsupported type: map[interface {}]interface {}error creating message for Grafana Loki: json: unsupported type: map[interface {}]interface {}error creating message for Grafana Loki: json: unsupported type: map[interface {}]interface {}[2019/04/15 13:59:32] [  Error] epoll_ctl: Bad file descriptor, errno=9 at /tmp/fluent-bit-1.0.6/lib/monkey/mk_core/mk_event_epoll.c:136
[2019/04/15 13:59:32] [error] [io_tls] error changing mask to 4
error creating message for Grafana Loki: json: unsupported type: map[interface {}]interface {}error creating message for Grafana Loki: json: unsupported type: map[interface {}]interface {}error creating message for Grafana Loki: json: unsupported type: map[interface {}]interface {}error creating message for Grafana Loki: json: unsupported type: map[interface {}]interface {}error creating message for Grafana Loki: json: unsupported type: map[interface {}]interface {}error creating message for Grafana Loki: json: unsupported type: map[interface {}]interface {}error creating message for Grafana Loki: json: unsupported type: map[interface {}]interface {}error creating message for Grafana Loki: json: unsupported type: map[interface {}]interface {}error creating message for Grafana Loki: json: unsupported type: map[interface {}]interface {}error creating message for Grafana Loki: json: unsupported type: map[interface {}]interface {}error creating message for Grafana Loki: json: unsupported type: map[interface {}]interface {}error creating message for Grafana Loki: json: unsupported type: map[interface {}]interface {}error creating message for Grafana Loki: json: unsupported type: map[interface {}]interface {}error creating message for Grafana Loki: json: unsupported type: map[interface {}]interface {}error creating message for Grafana Loki: json: unsupported type: map[interface {}]interface {}error creating message for Grafana Loki: json: unsupported type: map[interface {}]interface {}error creating message for Grafana Loki: json: unsupported type: map[interface {}]interface {}error creating message for Grafana Loki: json: unsupported type: map[interface {}]interface {}[2019/04/15 13:59:32] [  Error] epoll_ctl: Bad file descriptor, errno=9 at /tmp/fluent-bit-1.0.6/lib/monkey/mk_core/mk_event_epoll.c:136
[2019/04/15 13:59:32] [error] [io_tls] error changing mask to 4
```

**Comment by bavarianbidi** [comment]
@gregwebs 
i opened a PR to solving this issue (https://github.com/cosmo0920/fluent-bit-go-loki/pull/7)

we've done a loki-POC in the last week and during this POC, we didn't get the `unsupported type` error until we changed the json-parser

**Comment by cosmo0920** [comment]
Merged #7. Closing.
